gemrb/plugins/SDLVideo/SDL20Video.cpp

```diff
@@ -594,7 +594,11 @@ void SDL20VideoDriver::DrawRawGeometry(
 		nullptr,
 		vertices.data(),
 		2 * sizeof(float),
+		#if SDL_VERSION_ATLEAST(2, 0, 22)
 		reinterpret_cast<const SDL_Color*>(colors.data()),
+		#else
+		reinterpret_cast<const int*>(colors.data()),
```

Comment:
could you add this here: `static_assert(sizeof(int) == sizeof(SDL_Color));`

Reply:
I'm not sure I understand correctly. How do I put a block declaration in an argument list? You meant after or before SDL_RenderGeometryRaw() call?

Reply:
Done!
fad94a78f8c28bf01d64219aaf391f5561f3dfcd